src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -56,12 +56,19 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId, ..
 
   const formattedPrice = toFiat(marketData.price)
 
-  const { assetReference } = fromAssetId(asset.assetId)
-  const maybeToken = tokenOrUndefined(assetReference)
+  const href = (() => {
+    const { assetReference } = fromAssetId(asset.assetId)
+    const maybeToken = tokenOrUndefined(assetReference)
 
-  // If token is undefined, redirect to the basic explorer link
-  // else redirect to the token explorer link
-  const href = maybeToken ? `${asset?.explorerAddressLink}${maybeToken}` : asset?.explorer
+    if (isNft(asset.assetId)) {
+      const [token] = assetReference.split('/')
+      return `${asset.explorer}/token/${token}?a=${asset.id}`
```

Comment:
extend to add nft asset href (evm chains all share the same etherscan path for now)

Reply:
Happy with this implementation precisely since all EVM explorers are the same Etherscan copycats